README.md

```diff
@@ -4,6 +4,15 @@ Python bindings for the Domino API.
 
 Permits interaction with a Domino deployment from Python using the [Domino API](https://dominodatalab.github.io/api-docs/).
 
+## Version Compatibility Matrix
+
+`python-domino` is compatible to domino version as follows:
```

Comment:
Something's off about this sentence. `Domino` should be capitalized, at least. Here's how a similar thing is said in `play-pac4j`:

`Several versions of the library are available for the different versions of the Play framework:`